mass_mailing_partner/readme/DESCRIPTION.rst

```diff
@@ -0,0 +1,10 @@
+This module links mass-mailing contacts with partners.
+
+Features
+--------
```

Comment:
Isn't this gonna crash?

Reply:
Yeah, good catch. The only allowed subtitle is `~~~~~~`, and I'm not sure we should keep it.